### PR TITLE
docs: add page about OLTP

### PR DESCRIPTION
**Pull request by emschwartz**
- **docs: link to safety and performance docs**
- **docs: mention using multiple LSM trees in performance section**
- **docs: link from transfer reference to data modeling about amounts**
- **docs: add built for OLTP page**

### CODE REVIEW

File: docs/about/oltp.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+on everything it does, and makes heavy use of
+[assertions](https://tigerbeetle.com/blog/2023-12-27-it-takes-two-to-contract) and [simulation
+testing](https://tigerbeetle.com/blog/2023-07-06-simulation-testing-for-liveness) to check that code
+behaves exactly as expected.
+
+Importantly, TigerBeetle is one of the only OLTP databases that was designed to handle storage
+faults. Research on how storage faults affect databases only came out in 2018 (["fsyncgate
+2018"](https://danluu.com/fsyncgate/)), and TigerBeetle was built from the ground up with these
+lessons in mind. TigerBeetle uses cryptographic hash chains to detect storage faults, it supports
+[protocol-aware recovery](https://www.usenix.org/conference/fast18/presentation/alagappan), and uses
```
Comment (emschwartz) [inline docs/about/oltp.md]: Title case

File: docs/about/oltp.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+real-time. The control plane determines the network topology and makes decisions for how packets
+should be routed. The data plane must be as fast and efficient as possible, whereas the control
+plane does not need to be touched as frequently.
+
+TigerBeetle is designed to be in the hot path of your system, processing every transaction in
+real-time. You can have a [stateless API
+service](../develop/client-requests.md#example-api-layer-architecture) construct and batch transfers
+to send to TigerBeetle.
+
+Metadata, such as account details or infrequently changed ledger details, can be stored in a general
```
Comment (emschwartz) [inline docs/about/oltp.md:155]: Get rid of this section or add a better conclusion

File: docs/about/oltp.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+You could try to separate different groups of accounts into separate database clusters.
+Unfortunately, most accounting systems tend to have a few very hot accounts that are involved in a
+large number of transactions. For example, those representing the service operator's assets,
+liabilities, or income. You can shard accounts, but contention and row locks on the hot accounts
+become the bottleneck.
+
+Alternatively, certain types of very high-volume transaction systems like billing for serverless
+usage might use MapReduce to total the balances after the fact. However, this makes it extremely
+difficult to implement anything like real-time spending caps for or live account balance views for
```
Comment (emschwartz) [inline docs/about/oltp.md]:
```suggestion
difficult to implement anything like real-time spending caps or live account balance views for
```

**Review by jorangreef — DISMISSED** [review]
LATM!

**Review by jorangreef — APPROVED** [review]

**Pull request merged**